### PR TITLE
`database_to_json` Module Was Creating Datasets of Incorrect Format

### PR DESCRIPTION
**Pull request by epeters3**
Parent issue: #205.

It looks like the meta datasets `dna/database_to_json.py` was creating were of a little bit different format than the rest of the repo uses, e.g. different than the format of `small_classification.tar.xz` and `complete_classification.tar.xz`. This PR alters `dna/database_to_json.py` so it produces datasets of the correct format.

### CODE REVIEW

**Comment by epeters3** [comment]
This module operates upstream from any of the models, so this PR should be able to be reviewed independent of any discussion or code related to supporting unseen primitives and DAGs.

File: dna/database_to_json.py [hunk 1]
```diff
@@ -90,31 +99,45 @@ def get_pipeline_run_info(self, pipeline_run):
...
+        if test_score is None:
+            # This pipeline run was not scored using the supported metric for this
+            # problem type.
+            return None
+        dataset_id = pipeline_run["datasets"][0]["id"]
         test_predict_time = self.get_time_elapsed(pipeline_run)
-        train_accuracy = 0
-        train_predict_time = 0  # TODO have this find the fit pipeline and get the time
+        # TODO have this find the fit pipeline and get the train score
+        # and train predict time, then include those in the returned dictionary.
```
Comment (bjschoenfeld) [inline dna/database_to_json.py:120]: @epeters3 Can you open an issue for this?

Comment (bjschoenfeld) [reply dna/database_to_json.py:120]: *For this TODO

Comment (bjschoenfeld) [reply dna/database_to_json.py:120]: Actually, I think in D3M the scores are not computed on the training data.

**Review by bjschoenfeld — APPROVED** [review]

**Comment by bjschoenfeld** [comment]
@epeters3 Can you sync this with the base branch?

File: dna/database_to_json.py (hunk 1 above)
Comment (epeters3) [reply dna/database_to_json.py:120]: I think you're right. I'll open an issue for capturing just the train time, mentioning that that may be an interesting target for metalearning.

Comment (epeters3) [reply dna/database_to_json.py:120]: This has been documented in #208.

**Comment by epeters3** [comment]
The sync is done.

**Comment by epeters3** [comment]
Thanks for your comments @bjschoenfeld, ready again for your review.

**Review by bjschoenfeld — APPROVED** [review]

**Pull request merged**